recipes/kmodes/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "kmodes" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Can this be a `noarch` package?

Reply:
Sorry wrong line, this should have been in the build section. :sheep:

Reply:
Could you give me an example of other `noarch` packages ?
I always thought `noarch` was not recommended in conda.

Reply:
here are the noarch docs https://conda-forge.org/docs/meta.html#building-noarch-packages